LICENSE

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014-2017 Google Inc.
+   Copyright 2017 Google Inc.
```

Comment:
Is it possible to set this as `2014-${current}`?

Reply:
Will do: GoogleCloudPlatform/nodejs-repo-tools#22